spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+
+describe ReviewsController do
+  describe "POST create" do
+    context "for authenticated users" do
+      let(:video) { Fabricate(:video) } 
+      let(:current_user) { Fabricate(:user) }
+
+      before do
+        session[:user_id] = current_user.id 
+      end
+
+      context "with valid input" do
+        before do
+          post :create, review: Fabricate.attributes_for(:review), video_id: video.id, user_id: session[:user_id] 
+        end
+
+        it "redirects to video show template" do
+          expect(response).to redirect_to video_path(video)
```

Comment:
Redirecting to a path and rendering a template are two different things, but your spec title conflates them.